example/viewModeChangeExample.js

```diff
@@ -0,0 +1,34 @@
+var DateTime = require('../DateTime.js');
+var React = require('react');
+var ReactDOM = require('react-dom');
+var moment = require('moment');
+
+var Wrapper = React.createClass({
+	getInitialState: function() {
+		return {
+			viewMode: 'time'
+		};
+	},
+
+	updateView: function(format) {
+		console.log('changing state');
+		this.setState({
+			viewMode: 'days'
+		});
+	},
+
+	componentDidMount: function() {
+		setTimeout(this.updateView, 3000);
+	},
+
+	render: function() {
+		console.log('Current viewmode: ' + this.state.viewMode);
+		return React.createElement(DateTime,
+            { viewMode: this.state.viewMode, defaultValue: moment()	});
```

Comment:
Also, the indentation on this line is weird. It looks like it's been adjust in the wrong direction. Sorry for these nitpicks but I want a clean code base.